examples/text_embeddings_v2/export_test_v2.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2019 The TensorFlow Hub Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tests for text embedding exporting tool v2."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import logging
+import os
+from distutils.version import LooseVersion
+import numpy as np
+import tensorflow as tf
+import tensorflow_hub as hub
+
+from examples.text_embeddings_v2 import export_v2 as export
```

Comment:
Please do `from examples.text_embeddings_v2 import export_v2`, then use `export_v2` in code. No need to rename it.